lib/services/presets.go

```diff
@@ -29,6 +29,68 @@ import (
 	"github.com/gravitational/teleport/lib/modules"
 )
 
+func NewPresetAutomaticAccessApproverRole() types.Role {
+	enterprise := modules.GetModules().BuildType() == modules.BuildEnterprise
+	if !enterprise {
+		return nil
+	}
+	role := &types.RoleV6{
+		Kind:    types.KindRole,
+		Version: types.V7,
+		Metadata: types.Metadata{
+			Name:        teleport.PresetAutomaticAccessApprovalRoleName,
+			Namespace:   apidefaults.Namespace,
+			Description: "Approves any access request",
+			Labels: map[string]string{
+				types.TeleportResourceTypeLabel: string(types.TeleportResourceTypeSystem),
```

Comment:
It says "preset" everywhere (function name, constant name) but then the label is System.
That would be ok if there was no `preset` label for TeleportResourceTypeLabel, but there is.

So, can we please change the naming to use `System`? Otherwise, this seems confusing